src/field/vector2d.cxx

```diff
@@ -67,7 +65,9 @@ void Vector2D::toCovariant() {
     Mesh* localmesh = getMesh();
 
     if (location == CELL_VSHIFT) {
-      Coordinates *metric_x, *metric_y, *metric_z;
+      Coordinates* metric_x;
+      [[maybe_unused]] Coordinates* metric_y;
```

Comment:
Why is this `[[maybe_unused]]`? Marking a local variable `[[maybe_unused]]` should set off alarm bells in your head -- look at the diff of L91-94 below!